wdio.conf.js

```diff
@@ -8,94 +8,121 @@ const { getScreenshotName, getRefPicName, getDiffScreenshotName } = visualRegres
 let isDefaultBrowser = true;
 let sauceLabsUsername;
 let saucelabsAccesKey;
-let driver = 'selenium-standalone';
 let browsers = [];
+let driver = 'selenium-standalone';
+
+if (e2eProfile.includes('saucelight')) {
+  browsers.push({
+    browserName: 'chrome',
+    version: 'latest',
+    screenResolution: screenResolution,
+    platform: 'macOS 10.13'
 
-if (e2eProfile.includes('sauce')) {
+  });
+
+  driver = 'sauce';
   isDefaultBrowser = false;
   sauceLabsUsername = process.env.SAUCE_LABS_USERNAME;
   saucelabsAccesKey = process.env.SAUCE_LABS_ACCESS_KEY;
-  driver = 'sauce';
 
-  if (e2eProfile === 'saucelight') {
-    browsers = [{
-      browserName: 'chrome',
-      version: 'latest',
-      screenResolution: screenResolution,
-      platform: 'macOS 10.13'
-    }];
-  } else if (e2eProfile === 'sauceextended') {
-    browsers = [{
-      browserName: 'firefox',
-      version: 'latest',
-      screenResolution: screenResolution,
-      platform: 'macOS 10.13'
-    }, {
-      browserName: 'chrome',
-      version: 'latest',
-      screenResolution: screenResolution,
-      platform: 'macOS 10.13'
-    }, {
-      browserName: 'chrome',
-      'chromeOptions': {
-        'args': ['disable-infobars']
-      },
-      version: 'latest-1',
-      screenResolution: screenResolution,
-      platform: 'Windows 10'
-    }, {
-      browserName: 'firefox',
-      version: 'latest',
-      screenResolution: screenResolution,
-      platform: 'Windows 10'
-    }, {
-      browserName: 'chrome',
-      'chromeOptions': {
-        'args': ['disable-infobars']
-      },
-      version: 'latest',
-      screenResolution: screenResolution,
-      platform: 'Windows 10'
-    }, {
-      browserName: 'chrome',
-      version: 'latest-1',
-      screenResolution: screenResolution,
-      platform: 'macOS 10.13'
-    }];
-  }
-} else if (!e2eProfile.includes('sauce') && e2eProfile !== '') {
+}
+if (e2eProfile.includes('sauceextended')) {
+  browsers.push({
+    browserName: 'firefox',
+    version: 'latest',
+    screenResolution: screenResolution,
+    platform: 'macOS 10.13'
+  }, {
+    browserName: 'chrome',
+    version: 'latest',
+    screenResolution: screenResolution,
+    platform: 'macOS 10.13'
+  }, {
+    browserName: 'chrome',
+    'chromeOptions': {
+      'args': ['disable-infobars']
+    },
+    version: 'latest-1',
+    screenResolution: screenResolution,
+    platform: 'Windows 10'
+  }, {
+    browserName: 'firefox',
+    version: 'latest',
+    screenResolution: screenResolution,
+    platform: 'Windows 10'
+  }, {
+    browserName: 'chrome',
+    'chromeOptions': {
+      'args': ['disable-infobars']
+    },
+    version: 'latest',
+    screenResolution: screenResolution,
+    platform: 'Windows 10'
+  }, {
+    browserName: 'chrome',
+    version: 'latest-1',
+    screenResolution: screenResolution,
+    platform: 'macOS 10.13'
+  });
+
+  driver = 'sauce';
   isDefaultBrowser = false;
```

Comment:
you can check what is the driver and decide based on that whether it's the default browser or not